.github/ISSUE_TEMPLATE/0-bug-report-template.yml

```diff
@@ -0,0 +1,110 @@
+name: New Bug Report
+description: File a bug report
+title: "Product-Area-Here: short description of bug here"
+labels:
+  - bug
+  - needs triage
```

Comment:
I think that by adding a "needs triage" label to issues, it would help us easily filter to see new stuff that we need to look at (since non-grafana members cannot label the issues themselves)